code/game/gamemodes/cult/runes.dm

```diff
@@ -316,6 +319,12 @@ var/list/teleport_runes = list()
 		to_chat(user, "<span class='cult'>You[moveuserlater ? "r vision blurs, and you suddenly appear somewhere else":" send everything above the rune away"].</span>")
 		if(moveuserlater)
 			user.forceMove(get_turf(actual_selected_rune))
+		user.apply_damage(6, BRUTE)
+		var/mob/living/carbon/human/H = user
+		if(user.z != T.z)
```

Comment:
Maybe have extra damage for people who are teleporting to/from deep space? (IE: not station or mining zlevels)